test/test_driving_key_field.py

```diff
@@ -0,0 +1,33 @@
+"""Unit test for DrivingKeyField."""
+from diepvries import TABLE_PREFIXES, TableType
+from diepvries.driving_key_field import DrivingKeyField
+
+
+def test_satellite_name_prefix(ls_order_customer_eff_driving_keys: DrivingKeyField):
```

Comment:
I think we're not really testing the new code with these tests right? 

I'd expect the new tests to run the new version of the code, so possibly three tests:
- One checking that instantiating a `DrivingKeyField` with correct inputs does not fail
- One checking that instantiating a `DrivingKeyField` with a wrong parent link name fails
- One checking that instantiating a `DrivingKeyField` with a wrong satellite name fails

An example would be something like:
```
def test_validate_invalid_parent_table_name():
    with pytest.raises(RuntimeError):
        _=DrivingKeyField(parent_table_name="invalid_link", name="h_foo_hashkey", "ls_foo_bar")
```

This one would check if instantiating a `DrivingKeyField` with an invalid link name would fail with `RuntimeError`.

Does this make sense?